doc/src/conf.py

```diff
@@ -73,7 +73,7 @@
 exclude_patterns = ['build', 'Thumbs.db', '.DS_Store']
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+pygments_style = 'colorful'
```

Comment:
Is this change needed? I feel like many of the colors in this theme are kind of strange.

Reply:
As far as I can tell, the highlighting style we used to use just isn't there anymore, the default that replaced it was horrible, and this was the best one I could find. If anyone can figure out how to get the old style back, please let me know!

Reply:
ok!